osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -457,6 +466,18 @@ namespace renderkit {
         delete m_library.OpenGL;
     }
 
+    bool RenderManagerOpenGL::GetTimingInfo(size_t whichEye, RenderTimingInfo& info) {
+      if(m_toolkit.getRenderTimingInfo) {
+        OSVR_RenderTimingInfo renderTimingInfo;
```

Comment:
[Question] It seems odd that there's `OSVR_RenderTimingInfo` and also `RenderTimingInfo` structs that have the same members. It's probably outside the scope of this PR to reconcile that (and there may be good reasons why they both exist). But if the same type were used throughout, you could just pass the `renderTimingInfo` reference into the `getRenderTimingInfo()` function and not have to copy everything by hand.

Reply:
No real reason. RenderTimingInfo was there first, but it's in C++ land only. No reason to have two though, so we could consolidate them.